project.godot

```diff
@@ -665,6 +665,10 @@ locale_filter=[ 0, [  ] ]
 
 project/assembly_name="Super Mario 63 Redux"
 
+[network]
+
+limits/debugger_stdout/max_chars_per_second=2147483647
```

Comment:
Was this accidental? I don't see what relevance this bears. I also don't know what this does. :P

Reply:
That...should already be in master. It raises the debug output limit to its absolute maximum, so it's possible to print_debug(...)` every frame and the console doesn't lock up within a few seconds.